connectors/src/connectors/slack/index.ts

```diff
@@ -629,3 +636,51 @@ export async function retrieveSlackChannelsTitles(
 
   return new Ok(titles);
 }
+
+export const getSlackConfig: ConnectorConfigGetter = async function (
```

Comment:
Any reason to not use the function `getSlackConfig` that lives in `getSlackConfig.ts`?

Reply:
Forgot to remove this file. My bad.